tests/test_analysis_utils.py

```diff
@@ -85,21 +85,20 @@ def setUp(self):
 
         # image properties
         self.img_count = 10
-        self.dim_x = 15
-        self.dim_y = 20
-        self.noise_level = 1.0
-        self.variance = self.noise_level**2
-        self.p = PSF(0.5)
-
-        # create image set with single moving object
-        self.imlist = []
-        self.time_list = []
-        for i in range(self.img_count):
-            time = i / self.img_count
-            self.time_list.append(time)
-            im = LayeredImage(self.dim_x, self.dim_y, self.noise_level, self.variance, time, self.p, i)
-            self.imlist.append(im)
-        self.stack = ImageStack(self.imlist)
+        self.dim_x = 100
+        self.dim_y = 100
```

Comment:
This looks inconsistent with what was replaced. Before self.dim_x = 15, here self.dim_x = 100. Same for dim_y. 

Is this intentional?

Reply:
This is an intentional change so the fake data can be used by multiple tests. The second test needs the larger image. The first only needs a smaller one, but the larger one doesn't hurt.